docs/README.md

```diff
@@ -215,22 +215,22 @@ Some [overlays](#overlays) get their information from local data sources, such a
 
 Specify the folders the app searches for local data using your [App configuration settings](#app-configuration-settings). Local data that has been found on your device is listed by this tool. Any data supported by ArcGIS Runtime SDKs can be used, including:
 
-- [GeoPackage]
+- [Geopackage]
 - [Tile package] (TPK)
 - [Mobile geodatabase]
 - KML
```

Comment:
I feel like we might as well add a link to KML in the glossary as well: https://developers.arcgis.com/documentation/glossary/kml/

Reply:
Good suggestion - resolved with d0dae19